src/core/src/core_logic/MaintenanceWindow.py

```diff
@@ -59,9 +59,13 @@ def get_remaining_time_in_minutes(self, current_time=None, log_to_stdout=False):
 
         return remaining_time_in_minutes
 
-    def is_package_install_time_available(self, remaining_time_in_minutes=None):
+    def is_packages_install_time_available(self, remaining_time_in_minutes=None, number_of_packages = 1, reboot_manager=None):
```

Comment:
is_package_install_time_available is fine because it's sort of like "is_passenger_boarding_time_available" for say an aircraft boarding even though obviously it could mean evaluation for one or more passengers.